setup-ros.sh

```diff
@@ -127,15 +121,23 @@ pip3 install --upgrade \
 	nose \
 	pep8 \
 	pydocstyle \
-	pyparsing \
 	pytest \
 	pytest-cov \
 	pytest-mock \
 	pytest-repeat \
 	pytest-rerunfailures \
 	pytest-runner \
-	setuptools \
-	wheel
+	setuptools
+UBUNTU_VERSION=$(lsb_release -cs)
+case ${UBUNTU_VERSION} in
+    "noble")
+        ;;
+    *)
+		pip3 install --upgrade \
+			pyparsing \
+			wheel
+        ;;
+esac
```

Comment:
As part of #67, this list needs to be cleaned up and switched to use `apt` when possible.